pages/mesosphere/dcos/1.14/installing/production/deploying-dcos/installation/index.md

```diff
@@ -353,6 +362,11 @@ At this point your directory structure should resemble:
     ```
 
 3.  <A name="masterinstall"></A> Run the following commands on each of your master nodes in succession to install DC/OS using your custom build file:
+    * If created, copy the Pre-Shared Key to your master nodes at /root/.dcos-bootstrap-ca-psk
```

Comment:
On I only now realize this path `/root/.dcos-bootstrap-ca-psk`. We usually only touch `/opt/mesosphere` (immutable), `/var/lib/dcos...` (mutable, persistent) and `/run/dcos/...` (not persistent).

In view of that we should consider putting this somewhere under `/var/lib/dcos.`

Reply:
For the sake of not polluting other directories I agree that we should use `/opt/mesosphere/dcos-bootstrap-ca-psk` path.

Reply:
@mhrabovcin I would prefer `/var/lib/dcos` because `/opt/mesosphere/.." is supposed to be populated by the installer automatically only. 

In case of the Custom CA Certificate capability we have an analogue use case:

> For security reasons, the installer will not copy the private key from the bootstrap node to the master nodes. The private key associated with the custom CA certificate must be distributed manually to every DC/OS master node before starting the installation.

> The filesystem path for the private key file must be /var/lib/dcos/pki/tls/CA/private/custom_ca.key. The directory /var/lib/dcos/pki/tls/CA/private/ can be created manually with the following command before putting the file custom_ca.key in the directory on every DC/OS master node:

(https://docs.d2iq.com/mesosphere/dcos/1.11/security/ent/tls-ssl/ca-custom/)

Reply:
Created https://jira.mesosphere.com/browse/DCOS-57650 for this